app/features/analysis/cards/OneRM/tags/edit_tags/EditTagsModal.js

```diff
@@ -0,0 +1,383 @@
+// TODO: consider splitting this component into two different ones rather than using if statements everywhere
+
```

Comment:
- [ ] instead of using the word edit, use select? SelectTagsModal for example
- [ ] don't use a folder called edit_Tags